cmd/libsnap-confine-private/utils-test.c

```diff
@@ -99,6 +99,18 @@ static void test_die_with_errno(void)
 	g_test_trap_assert_stderr("death message: Operation not permitted\n");
 }
 
+// A variant of rmdir that is compatible with GDestroyNotify
+static void my_rmdir(const char *path)
+{
+  rmdir(path);
```

Comment:
meta-question: should we log/error here if this (and chdir and unsetenv) return != 0 ? Given that this is tests only a error here might be ok?